NER_Data_collection_and_annotation.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>Nitpick, but most Python style guides will ask to leave a white space between assignment operators (except when it's being used to assign a default value to a function argument). </p><p> </p><p>E.g. <code>FIRST_RUN = True</code></p><p> </p><p>It's hard to enforce style guides in notebooks, so no need to change anything in that regards </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meetnaren/ner-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Got it! :thumbs up:</p>